tutorials/terraform/sqoop/clusters-postgresql-data-proc-and-vm.tf

```diff
@@ -0,0 +1,257 @@
+# Infrastructure for Yandex Cloud Managed Service for PostgreSQL cluster.
+#
+# RU: https://cloud.yandex.ru/docs/managed-postgresql/tutorials/sqoop
+#
+# Set the configuration of the Managed Service for PostgreSQL cluster, Managed Service for Data Proc cluster and Virtual machine:
```

Comment:
Set the configuration of the Managed Service for PostgreSQL cluster, Managed Service for Data Proc cluster, and Virtual machine: